snuba/snuba_migrations/replays/0014_add_id_event_columns.py

```diff
@@ -0,0 +1,43 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import UUID, Column
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+from snuba.migrations.operations import OperationTarget
+
+LOG_LEVELS = ["fatal", "error", "warning", "info", "debug"]
+
+
+class Migration(migration.ClickhouseNodeMigration):
+    blocking = False
+
+    def forwards_ops(self) -> Sequence[operations.SqlOperation]:
+        return [
+            operations.AddColumn(
+                storage_set=StorageSetKey.REPLAYS,
+                table_name=table_name,
+                column=Column(log_level + "_id", UUID()),
```

Comment:
Are these guaranteed to always exist? Shouldn't this be a Nullable column?

Reply:
I believe that we want to get a way from making nullable columns b.c. of performance -- cc @cmanallen. https://stackoverflow.com/questions/63057886/clickhouse-string-field-disk-usage-null-vs-empty

Reply:
Yeah I think its fine to leave as not null.  UUIDs support the `empty` and `notEmpty` functions.